src/common.jl

```diff
@@ -5,6 +5,20 @@
     POLYNOMIAL = 2
 end
 
+@enum AngleUnit begin
+    DEGREES
+    RADIANS
+end
+
+@enum BusType begin
```

Comment:
Note this change.  The code now uses this instead of strings.  It should lead to better robustness, but it could also break some existing code. Let me know if you disagree.

Also, note the comment below about "ISOLATED"